base/multi.jl

```diff
@@ -813,7 +813,7 @@ function lookup_ref(pg, rrid, f)
         rv = get(pg.refs, rrid, false)
         if rv === false
             # first we've heard of this ref
-            rv = RemoteValue(f())
+            rv = RemoteValue(eval(Main, Expr(:call, f)))
```

Comment:
It's a rarely-used API, but comes up in test/example.jl due to `dc=RemoteChannel(()->DictChannel(), 1)`